### ISSUE TITLE
comments at end of line are not stripped

### ISSUE THREAD

**Issue by jayvdb** [jayvdb/flake8-putty#12]
Currently a `#` comment is only removed if it is one a separate line.

i.e. A rule like

```
    tests/textlib_tests.py,/self.assert.*{{/ : +P103 # invalidly detected as {} format string
```

will add `P103 # invalidly detected as {} format string` as one of the codes to ignore, instead of ignoring  `P103`.

The stripping needs to done in such as way that `#` is possible in the regex.

**Comment by jayvdb** [comment]
At the very least, it should raise an error

**Comment by jayvdb** [comment]
Fixed with 1220229
